sklearn/metrics/pairwise.py

```diff
@@ -1862,7 +1862,7 @@ def pairwise_distances(
     valid scipy.spatial.distance metrics), the scikit-learn implementation
     will be used, which is faster and has support for sparse matrices (except
     for 'cityblock'). For a verbose description of the metrics from
-    scikit-learn, see the __doc__ of the sklearn.pairwise.distance_metrics
+    scikit-learn, see :func:`sklearn.metrics.pairwise.pairwise_distances`
```

Comment:
I think the original intend is to link to  `distance_metrics`:

```suggestion
    scikit-learn, see :func:`sklearn.metrics.pairwise.distance_metrics`
```

BTW This is already the `pairwise_distances` docstring, so linking to itself is a bit circular.

Reply:
Ah sloppy indeed I will refactor